### ISSUE TITLE
Bug: Navigating into item does not work with arrow keys

### ISSUE THREAD

**Issue by SaiShan** [SAP/fundamental-styles#86]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
The user should be able to navigate (arrow keys) and select (enter / space) entry using the keyboard. 

- Mega menu
- Menu
- Multi input
- Pagination
- Popover
- Search input
- Shell bar
- Side navigation

**Comment by droshev** [comment]
most of the components have been refactored
